scripts/deploy/optimisticTimelockDeploy.ts

```diff
@@ -0,0 +1,28 @@
+import { DeployUpgradeFunc } from '../../types/types';
+import { ethers } from 'hardhat';
+
+const fourDays = 4 * 24 * 60 * 60;
+
+const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const { core } = addresses;
+
+  const adminAddress = process.env.ADMIN_ADDRESS;
```

Comment:
Just in case lets make the env names specific to the contract so theres no collision. TIMELOCK_ADMIN_ADDRESS or something